target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -222,6 +222,7 @@ tplink,tl-wr941-v4)
 	ucidef_set_led_switch "lan3" "LAN3" "tp-link:green:lan3" "switch0" "0x08"
 	ucidef_set_led_switch "lan4" "LAN4" "tp-link:green:lan4" "switch0" "0x10"
 	;;
+tplink,tl-mr3420-v2|\
```

Comment:
This breaks alphabetical order. The entire case should be moved above tplink,tl-wr740n-v1.

Reply:
TL-MR3420 follows tl-wr841-v8 and tl-wr842n-v2 the LED sequence matters

Reply:
Can you edit the blocks commits? If I were to do it I will need to do another pull request